application/templates/cms/create_table_2.html

```diff
@@ -53,7 +53,7 @@ <h2 class="heading-medium no-top-border builder-section-number">Data</h2>
                   <form onsubmit="return false;">
                           <textarea class="form-control" id="data_text_area" rows="10"
                                     {% if form_disabled %}disabled="disabled"{% endif %}></textarea>
-                      <button type="button" class="btn button btn-primary" id="confirm-data">OK</button>
+                      <button type="button" class="btn button btn-primary" id="confirm-data">Next</button>
```

Comment:
But I only just changed this from "Okay"!

Reply:
Next!